src/test/mir-opt/const_prop/read_immutable_static.rs

```diff
@@ -10,10 +10,10 @@ fn main() {
 // START rustc.main.ConstProp.before.mir
 //  bb0: {
 //      ...
-//      _3 = const Scalar(alloc0+0) : &u8;
+//      _3 = const {alloc0+0: &u8};
 //      _2 = (*_3);
 //      ...
-//      _5 = const Scalar(alloc0+0) : &u8;
+//      _5 = const {alloc0+0: &u8};
```

Comment:
We should probably make sure that when printing out some MIR we include all of the referenced allocations as well. Please make an issue and add FIXMEs in every file that mentions miri `alloc`s anywhere (also, MIR tests should include the definitions of `alloc`s).